cves/kernel/CVE-2016-3137.yml

```diff
@@ -225,10 +224,10 @@ subsystem:
 
     Can be multiple subsystems involved, in which case you can make it an array
     e.g.
-        name: ["subsystemA", "subsystemB"] # ok
+        name: ['subsystemA', 'subsystemB'] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: 'serial'
```

Comment:
maybe add the subsystems of "drivers" and "USB" as well

Reply:
As serial is a part of those subsystems I feel my answer is adequate as is.